spec/helpers/application_helper_spec.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
-require 'rails_helper'
```

Comment:
What happened with these deleted 'rails_helper' lines?

Reply:
I read that you only need `spec_helper` during specs.  `rails_helper` is implicitly part of that `require`.  Correct me if you know otherwise.

Reply:
[`rails_helper`](https://github.com/sanger/sequencescape/blob/390611cc6aa1bdc468d78469c0079c175c6a8e9c/spec/rails_helper.rb#L4) imports `spec_helper`.